impeller/entity/contents/atlas_contents.cc

```diff
@@ -85,59 +64,54 @@ bool AtlasContents::Render(const ContentContext& renderer,
     return true;
   }
 
-  using VS = AtlasFillVertexShader;
-  using FS = AtlasFillFragmentShader;
+  auto& host_buffer = pass.GetTransientsBuffer();
 
-  const auto texture_size = texture_->GetSize();
-  if (texture_size.IsEmpty()) {
-    return true;
-  }
+  if (geometry_->GetVertexType() == GeometryVertexType::kColor) {
```

Comment:
So the idea is, since the color is the dst, draw that first, then draw the image with a configured blend mode. Something like that.